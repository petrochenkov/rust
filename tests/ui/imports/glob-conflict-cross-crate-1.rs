//@ aux-build:glob-conflict.rs

extern crate glob_conflict;

fn main() {
    glob_conflict::f(); //~ ERROR `f` is ambiguous
    glob_conflict::glob::f(); //~ ERROR `f` is ambiguous
}
