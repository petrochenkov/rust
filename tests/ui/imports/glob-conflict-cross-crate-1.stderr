error[E0659]: `f` is ambiguous
  --> $DIR/glob-conflict-cross-crate-1.rs:6:20
   |
LL |     glob_conflict::f();
   |                    ^ ambiguous name
   |
   = note: ambiguous because of multiple glob imports of a name in the same module
note: `f` could refer to the function defined here
  --> $DIR/auxiliary/glob-conflict.rs:10:9
   |
LL | pub use m1::*;
   |         ^^
note: `f` could also refer to the function defined here
  --> $DIR/auxiliary/glob-conflict.rs:11:9
   |
LL | pub use m2::*;
   |         ^^

error[E0659]: `f` is ambiguous
  --> $DIR/glob-conflict-cross-crate-1.rs:7:26
   |
LL |     glob_conflict::glob::f();
   |                          ^ ambiguous name
   |
   = note: ambiguous because of multiple glob imports of a name in the same module
note: `f` could refer to the function defined here
  --> $DIR/auxiliary/glob-conflict.rs:10:9
   |
LL | pub use m1::*;
   |         ^^
note: `f` could also refer to the function defined here
  --> $DIR/auxiliary/glob-conflict.rs:11:9
   |
LL | pub use m2::*;
   |         ^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0659`.
